utilities/post_migration.py

```
@@ -1119,6 +1224,44 @@ def check_vms(
+        # Check node placement if configured
+        if plan.get("target_node_selector"):
+            if labeled_worker_node is None:
+                res[vm_name].append("check_vm_node_placement - labeled_worker_node fixture not provided")
+            else:
+                expected_node = labeled_worker_node.get("node_name")
+                if expected_node is None:
+                    res[vm_name].append("check_vm_node_placement - labeled_worker_node missing 'node_name' key")
+                else:
+                    try:
+                        check_vm_node_placement(destination_vm=destination_vm, expected_node=expected_node)
+                    except Exception as exp:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)